fs/operations/multithread.go

```diff
@@ -312,8 +312,8 @@ func (w *writerAtChunkWriter) WriteChunk(ctx context.Context, chunkNumber int, r
 	// if we were buffering, flush to disk
 	switch w := writer.(type) {
 	case *bufio.Writer:
-		er2 := w.Flush()
-		if er2 != nil {
+		err := w.Flush()
```

Comment:
```suggestion
		err = w.Flush()
```

A small additional adjustment is necessary, or else build with fail with:

> no new variables on left side of :=